Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
       capistrano (~> 3.1)
-    capistrano-rails (1.5.0)
+    capistrano-rails (1.6.1)
```

Comment:
### [capistrano-rails](https://github.com/capistrano/rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.5.0 → 1.6.1

[_[change-log](https://github.com/capistrano/rails/releases), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/capistrano/rails/compare/v1.5.0...v1.6.1).

These are the individual commits:
- (d8c547a) [Add migration_command accessor for overriding (#242)](https://github.com/capistrano/rails/commit/d8c547a8fe93b7a7653dc636e7cfae9e45262fc0)
- (e44e9c8) [Maintain release notes using GitHub Actions (#243)](https://github.com/capistrano/rails/commit/e44e9c8accb5716b3f2d337330fbe596a11901e8)
- (7d19e7c) [Use the latest version of capistrano-bundler (#244)](https://github.com/capistrano/rails/commit/7d19e7c4935722e3df507b98b3ed969361c1f599)
- (7e41411) [Release 1.6.0](https://github.com/capistrano/rails/commit/7e4141113b5c4917044c06910674b2ac8cf6c244)
- (c2f84da) [Fix SystemStackError when running deploy:migrate (#247)](https://github.com/capistrano/rails/commit/c2f84da90b59ef3a13d75208192163ed1a80b04c)
- (082255a) [Release 1.6.1](https://github.com/capistrano/rails/commit/082255af014d9d3f777489faa1081d6c8e3f74d0)


</details>